### ISSUE TITLE
Venison Stew

### ISSUE THREAD

**Issue by whichonespink44** [whichonespink44/Appalachia#161]
Recipe = venison (raw or cooked), potatoes and carrots.

Some preliminary discussion can found in #156.

**Comment by garantiertnicht** [comment]
Is the recipe going to work with booth, or is it not-decided-yet?

**Comment by whichonespink44** [comment]
I think both is fine, but happy to go with consensus.

**Comment by garantiertnicht** [comment]
If we allow booth, it should be a difference. Else noone would make the cooked variant for the stew.

**Comment by whichonespink44** [comment]
Good point... maybe we should restrict it to cooked venison only, which would then justify the additional saturation effect.

**Comment by srs-bsns** [comment]
This was only meant to be an initial idea, which is why I added it to the dear issue (as a reminder) instead of opening a separate issue for it.

Whether the recipe requires raw or cooked venison was more of a question, since there's always the choice of using raw to make the stew (raw) and then cooking the stew.

What's important though, is that the raw and cooked venison are added to the ore dictionary properly to match the equivent entries from other mods, like Pam's Harvestcraft. In the case that another mod is present that adds a recipe for a similar item (stew) then we can decide if we even want to register our stew item and it's recipe in those situations.

It's probably best to finish the mobs, and any items they will drop, first, before moving on to more refined parts of APL such as recipes & crafted items.

Also, my suggestion for the stew giving `regeneration I` for a short period stands. There are mods that remove the vanilla regeneration effect from saturation because it's kinda OP. I'm pretty sure Tough As Nails removes it. This way the stew would keep the effect. But this was also just an initial idea.
